week4/model_comparison.ipynb

```diff
@@ -0,0 +1,363 @@
+{
```

Comment:
can we throw these into a `post_session` subfolder? I know we haven't been consistent about this, but could help with clarity. Perhaps also tailor the readme file to explain the contents like we have done for at least some of the previous sessions